dart/lib/src/event_processor/enricher/io_enricher_event_processor.dart

```diff
@@ -112,7 +112,8 @@ class IoEnricherEventProcessor implements EnricherEventProcessor {
   SentryDevice _getDevice(SentryDevice? device) {
     final platformMemory = PlatformMemory(_options);
     return (device ?? SentryDevice()).copyWith(
-      name: device?.name ?? Platform.localHostname,
+      name: device?.name ??
```

Comment:
until now by default if we didn't get a device name from the native sdks, we defaulted to using the localHostname api